packages/block-library/src/heading/edit.js

```diff
@@ -80,7 +80,7 @@ function HeadingEdit( {
 				onReplace={ onReplace }
 				onRemove={ () => onReplace( [] ) }
 				aria-label={ __( 'Heading text' ) }
-				placeholder={ placeholder || __( 'Write heading…' ) }
+				placeholder={ placeholder || __( 'Heading' ) }
```

Comment:
Why not remove "add" from other placeholders?

Reply:
I wanted to keep it on fields like caption or citation, which are optional. The Quote is the odd one out at the moment which we should probably align